core/assets/bundles/bundle_nl.properties

```diff
@@ -194,68 +194,68 @@ save.delete.confirm = Weet je zeker dat je dit opgeslagen spel wilt verwijderen?
+save.corrupted = [accent]Bestand beschadigd of ongeldig!\nAls je je spel net hebt bijgewerkt, is dit waarschijnlijk een wijziging in het opslagformaat en [scarlet]geen[] bug.
 empty = <empty>
-on = On
-off = Off
-save.autosave = Autosave: {0}
+on = Aan
+off = Uit
+save.autosave = Automatisch opslaan: {0}
 save.map = Map: {0}
-save.wave = Wave {0}
-save.mode = Gamemode: {0}
+save.wave = Golf {0}
+save.mode = Spelmodus: {0}
 save.date = Last Saved: {0}
-save.playtime = Playtime: {0}
-warning = Warning.
-confirm = Confirm
-delete = Delete
-view.workshop = View In Workshop
-workshop.listing = Edit Workshop Listing
-ok = OK
+save.playtime = Speeltijd: {0}
+warning = Waarschuwing.
+confirm = Bevestigen
+delete = Verwijderen
+view.workshop = In Workshop bekijken
+workshop.listing = Workshop lijst bewerken
+ok = Oke
 open = Open
-customize = Customize
-cancel = Cancel
+customize = Personaliseren
+cancel = Annuleren
 openlink = Open Link
-copylink = Copy Link
-back = Back
-data.export = Export Data
-data.import = Import Data
-data.exported = Data exported.
-data.invalid = This isn't valid game data.
-data.import.confirm = Importing external data will erase[scarlet] all[] your current game data.\n[accent]This cannot be undone![]\n\nOnce the data is imported, your game will exit immediately.
-classic.export = Export Classic Data
-classic.export.text = [accent]Mindustry[] has just had a major update.\nClassic (v3.5 build 40) save or map data has been detected. Would you like to export these saves to your phone's home folder, for use in the Mindustry Classic app?
-quit.confirm = Are you sure you want to quit?
-quit.confirm.tutorial = Are you sure you know what you're doing?\nThe tutorial can be re-taken in[accent] Settings->Game->Re-Take Tutorial.[]
-loading = [accent]Loading...
-reloading = [accent]Reloading Mods...
-saving = [accent]Saving...
-cancelbuilding = [accent][[{0}][] to clear plan
-selectschematic = [accent][[{0}][] to select+copy
-pausebuilding = [accent][[{0}][] to pause building
-resumebuilding = [scarlet][[{0}][] to resume building
-wave = [accent]Wave {0}
-wave.waiting = [LIGHT_GRAY]Wave in {0}
-wave.waveInProgress = [LIGHT_GRAY]Wave in progress
-waiting = [LIGHT_GRAY]Waiting...
-waiting.players = Waiting for players...
-wave.enemies = [LIGHT_GRAY]{0} Enemies Remaining
-wave.enemy = [LIGHT_GRAY]{0} Enemy Remaining
-loadimage = Load Image
-saveimage = Save Image
-unknown = Unknown
+copylink = Kopieer Link
+back = Terug
+data.export = Exporteer Data
+data.import = Importeer Data
+data.exported = Data geëxporteerd.
+data.invalid = Dit is geen geldige spel data.
+data.import.confirm = Door externe gegevens te importeren, worden [scarlet] al[] je huidige spelgegevens gewist.\n[accent]Dit kan niet ongedaan worden![]\n\nZodra de gegevens zijn geïmporteerd, wordt je spel onmiddellijk afgesloten.
+classic.export = Exporteer klassieke data
```

Comment:
Suggestion (No change required):
Classic refers to the name of the stand-alone and publicly available Mindustry Classic (A legacy version), I suggest not translating it but using "Classic-Data" instead. (This issue arises in multiple lines.)

Reply:
I'll change it